charter.html

```diff
@@ -233,44 +237,40 @@ <h2 id=proposals>Proposals</h2>
 <p>The Community Group may discuss the Proposal on GitHub and during
 teleconferences or face-to-face <a href=#meetings>meetings</a>.
 
-<p>The <a href=#chairs>Chairs</a> may create a dedicated repository for
-a Proposal. They may do this for any reason, such as but not limited to:
+<p>Champions are responsible for the technical content of their
+Proposal. They are expected to solicit input from Community Group
+Participants, and they may consider and respond to comments,
+suggestions, and objections on their Proposal from Participants and the
+public.
 
-<ul>
-<li>because the Proposal has become sufficiently complex that writing an
-explainer is warranted
-<li>because the Proposal is sufficiently complex that having a dedicated
-issue tracker for it is warranted
-<li>because multiple <a href=#implementer>implementers</a> have
-expressed interest in the Proposal
-</ul>
+<p>Champions may ask the <a href=#chairs>Chairs</a> to create a
+dedicated repository for their Proposal.
```

Comment:
Technically, this says they Champions may ask, but it doesn't say anything about what the Chairs do in response. Maybe this is too nitpick, but I think it would help clarity to say whether the chairs may do it, must do it, should do it unless there is good reason not to, or something else.

Reply:
I added text in 7ff0490 that hopefully addresses this concern.